### ISSUE TITLE
Auto completion doesn't work correctly after switch case statement with lambda expression

### ISSUE THREAD

**Issue by Gaoyanei** [redhat-developer/vscode-java#1922]
The auto completion can't auto complete classes‘ names for lines in and after the lambda expression.

##### Environment
- Operating System: Windows 10
- JDK version:  OpenJDK 15.0.1
- Visual Studio Code version: 1.55.2
- Java extension version: 0.79.0

My code:
```
public class Aff {
    public static void main(String[] args) {
        try {
            switch (args[0]) {
                case "value":
                    System.out.println(0);
                    break;

                default:
                    System.out.println(1);
                    break;
            }

        } catch (Exception e) {
            e.printStackTrace();
        }
        Runnable r1 = () -> {
            // Auto completion doesn't work correctly from this line.
        };
    }
}
```

##### Current Result

![image](https://user-images.githubusercontent.com/83591686/116883490-3f0f5100-ac58-11eb-8c08-09b516c4c199.png)
![image](https://user-images.githubusercontent.com/83591686/116886624-e0e46d00-ac5b-11eb-87fd-a9a5c8eb7cce.png)
![image](https://user-images.githubusercontent.com/83591686/116886198-5865cc80-ac5b-11eb-8f23-e1772b27a1ad.png)

**Comment by jdneo** [comment]
This should be fixed in latest version. @testforstephen

**Comment by testforstephen** [comment]
Close the issue since it has been fixed in 1.3.0. @Gaoyanei pls have a try on the latest version.
